README.md

```diff
@@ -11,9 +11,9 @@ extern crate unicode_segmentation;
 use unicode_segmentation::UnicodeSegmentation;
 
 fn main() {
-    let s = "a̐éö̲\r\n";
```

Comment:
Please revert the sample string changes. Your editor seems to have applied NFC normalization to the file.

Reply:
oopsy - that's a issue w the github web editor.